.mvn/extensions.xml

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<extensions>
-    <extension>
-        <groupId>io.takari.maven</groupId>
-        <artifactId>takari-smart-builder</artifactId>
```

Comment:
Should you also remove `-b smart` from Maven calls in `.github/workflows/ci.yml`?

Reply:
Btw we should be fine with removing this, I tested it in https://github.com/trinodb/trino/pull/20485 and haven't noticed any significant changes in CI job times.

Reply:
indeed! thanks